### ISSUE TITLE
fix Argo reindex route to use dor_indexing_app

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/dor_indexing_app#105]
see #60. basically, we need to replace the `reindex_pid` call in Argo with a method that makes a synchronous request to the dor_indexing_app's API. same with the `delete_from_index` API.

- [x] https://github.com/sul-dlss/argo/blob/master/app/controllers/dor/objects_controller.rb#L20
- [x] https://github.com/sul-dlss/argo/blob/master/app/controllers/dor_controller.rb#L26-L27
- [x] https://github.com/sul-dlss/argo/blob/master/app/controllers/dor_controller.rb#L48-L50

- [x] deploy dor-indexing-app-dev to match argo-dev
- [x] deploy dor-indexing-app-stage to match argo-stage
- [ ] deploy dor-indexing-app-prod to match argo-prod

**Comment by drh-stanford** [comment]
See https://github.com/sul-dlss/argo/pull/891

**Comment by drh-stanford** [comment]
Blocked on https://github.com/sul-dlss/dor_indexing_app/issues/113

**Comment by drh-stanford** [comment]
This is in argo-stage ready for acceptance testing

**Comment by LynnMcRae** [comment]
In testing the behavior (looks good so far) I came across this anomaly relating to the indexing itself.  See screen shots ....
- before: no License facet, 3 "No license" entries under Empty Fields (as expected
- after: License facet with one blank entry, 2  "No license" entries under Empty Fields.

The odd item out is druid:wq307yk9043 with no rightsMetadata datastream.

**Comment by LynnMcRae** [comment]
![screen shot 2016-11-28 at 12 23 41 pm](https://cloud.githubusercontent.com/assets/1824732/20684890/dd10a86a-b566-11e6-84bd-e5a1b0836012.png)
![screen shot 2016-11-28 at 12 24 15 pm](https://cloud.githubusercontent.com/assets/1824732/20684893/e1539c0c-b566-11e6-9d2a-6317ce7ed094.png)

**Comment by LynnMcRae** [comment]
Trying a large batch (which may or may not be relevant) ... getting all errors
![screen shot 2016-11-28 at 12 50 36 pm](https://cloud.githubusercontent.com/assets/1824732/20685469/6caf5e10-b569-11e6-8e0b-2727ab0abe35.png)

**Comment by LynnMcRae** [comment]
individual blue button reindex for the first few items above also failing, but items from other collections seem to work ok.  This is a Revs collection with it's collections within collections ... likely data related (-stage related?).  Looking to see if there's anything obvious.

**Comment by LynnMcRae** [comment]
Blank entry in License facet is not new behavior.  Lots of entries like this in prod.  Will do a separate ticket.
